chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,869 @@
+    }
+
+    return url_in + add + var_in;
+  }
+
+  // Callback methods for replacer
+  replacer_onEach(text, re, new_text, value, key) {
+    re = new RegExp(`%${this.escapeRegExp(key)}%`, 'gi');
+    new_text = text.replace(re, value);
+    text = new_text;
+  }
+
```

Biome (1.9.4)

[error] Duplicate class member name "replacer_onEach"



(lint/suspicious/noDuplicateClassMembers)